mirrord/intproxy/src/proxies/incoming.rs

```diff
@@ -126,7 +126,6 @@ struct HttpGatewayHandle {
 /// An HTTP request stolen with a filter can result in an HTTP upgrade.
 /// When this happens, the TCP connection is recovered and passed to a new [`TcpProxyTask`].
 /// The TCP connection is then treated as stolen without a filter.
-#[derive(Default)]
```

Comment:
We can also remove `Default` from `ClientStore` to just to prevent any confusion with `IDLE_CLIENT_DEFAULT_TIMEOUT`